src/gridfs/file.rs

```diff
@@ -192,6 +192,7 @@ impl File {
                 }
                 self.doc.md5 = self.wsum.result_str();
                 try!(self.gfs.files.insert_one(self.doc.to_bson(), None));
+                try!(self.gfs.chunks.create_index(doc!{ "files_id" => 1, "n" => 1}, None));
```

Comment:
Context: the  mongo tools (like mongofiles, mongoimport) in versions before 3.0 were built with the mongo binaries. But >= v3.0 has "relegated" the tools (including their accompanying files like gridfs.{h/cpp} to the mongo tools) repo.